pkg/registry/core/service/storage/rest_test.go

```diff
@@ -262,7 +262,7 @@ func NewTestRESTWithPods(t *testing.T, endpoints *api.EndpointsList, pods *api.P
 }
 
 func makeIPNet(t *testing.T) *net.IPNet {
-	_, net, err := net.ParseCIDR("1.2.3.0/24")
```

Comment:
is makeIPNet6 good?

Reply:
it has a /108 that are 2^24 , we have it limited to 2^20 IIRC, should not be a problem if we don't hit it now with 2^16